lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -0,0 +1,22 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      class BuiltinMethods
+        def self.email(params = {})
+          options = {
+            :to      => params["To"],
+            :from    => params["From"],
+            :subject => params["Subject"],
+            :cc      => params["Cc"],
+            :bcc     => params["Bcc"],
+            :body    => params["Body"]
+          }
```

Comment:
We can make the interface more forgiving with something like:

```suggestion
          options = params.transform_keys { |k| k.to_s.downcase.to_sym }.slice(*%i[to from subject cc bcc body])
```

Reply:
There might be a nicer way to do that, but you get the idea :)  - whitelist first for example might be better:
```suggestion
          options = params.select { |k, _v| %w[to from subject cc bcc body].include?(k.to_s.downcase) }.transform_keys { |k| k.to_s.downcase.to_sym }
```